recipes/octo-encryption-cpp/all/conanfile.py

```diff
@@ -0,0 +1,97 @@
+from conans import ConanFile
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+from conan.tools.files import get
+from conan.tools.build import check_min_cppstd
+from conans.errors import ConanInvalidConfiguration
+from conan.tools.scm import Version
+
+
```

Comment:
```suggestion
required_conan_version = ">=1.50.0"


```
Conan will parse and check conan client is valid

Reply:
done